packages/slice-machine/components/ToasterContainer/index.tsx

```diff
@@ -23,6 +24,8 @@ const getIconAccordingToasterType = ({
       return <MdDone />;
     case toast.TYPE.WARNING:
       return <MdWarning />;
+    default:
+      return false;
```

Comment:
if it's JSX.Element returned, this should most likely return Null

Reply:
The icon prop of `ReactToastifyContainer` takes either the react element or false (for no icon), hence why I went with this option. I agree that looking at this function looks weird though so I'll go with the null option

Reply:
ah ok 👍 , well your solution was well so my bad, I should have asked why you decided to go with that.